python/pyspark/ml/classification.py

```diff
@@ -93,9 +93,7 @@ def __init__(self, featuresCol="features", labelCol="label", predictionCol="pred
         self._java_obj = self._new_java_obj(
             "org.apache.spark.ml.classification.LogisticRegression", self.uid)
         #: param for threshold in binary classification, in range [0, 1].
-        self.threshold = Param(self, "threshold",
-                               "Threshold in binary classification prediction, in range [0, 1]." +
-                               " If threshold and thresholds are both set, they must match.")
+        self.threshold = LogisticRegression.threshold._copy_new_parent(self)
```

Comment:
```LogisticRegression.threshold``` is a placeholder to make the param appear in the generated doc. After this change, it has more uses, should we also change the annotation of that dummy param?